src/network/networkd-link.c

```diff
@@ -3076,6 +3079,9 @@ static int link_carrier_lost(Link *link) {
 
         assert(link);
 
+        if (link->state == LINK_STATE_CONFIGURED && link->network->configure_without_carrier)
+                return 0;
+
         r = link_stop_clients(link);
```

Comment:
Don't we want to stop the clients as there is no carrier. When the carrier will gain again it will start the clients.

Reply:
what does link_stop_clients do? My idea was that once the link is configured then do nothing at all on carrier changes. And hence IgnoreCarrier.

Reply:
what does link_stop_clients do? My idea was that once the link is configured then do nothing at all on carrier changes. And hence IgnoreCarrier.